src/components/Modal/Modal.styles.ts

```diff
@@ -3,8 +3,24 @@ import colors from '../../styles/colors';
 import fonts from '../../styles/fonts';
 import { ModalProps } from './types';
 
-type TStyleProps = Pick<ModalProps, 'isVisible' | 'hasCancel' | 'width'>;
+type TStyleProps = Pick<
+    ModalProps,
+    'isVisible' | 'hasCancel' | 'width' | 'canOverflow'
+>;
 
+const overflow = (v: boolean) => {
+    console.log(v);
+    return v
+        ? ''
+        : `
+                overflow: auto;
+                ::-webkit-scrollbar {
+                    display: none;
+                }
+                scrollbar-width: none;
+                -ms-overflow-style: none;
+            `;
+};
```

Comment:
console.log

maybe its better to return the css after an if statement so we dont return an empty string;

`if(!v) return css`